blues/airnote.live
# deps: update sveltekit version to v2

# Problem Context

SvelteKit 2.0 debuted recently so we should change with the times and upgrade our Airnote project to keep pace.

## Changes

Ran the SvelteKit-provided codemod to update all necessary dependencies to SvelteKit v2 (included deps in package.json, and a few minor code changes to +page.server.ts files)

## Screenshot (if applicable)

n/a UI is unchanged

## Testing

Unit / integration / e2e tests?

All tests pass

Steps to test manually?

1. Go to deploy preview and add Airnote device UID to URL like `https://deploy-preview-66--airnote-live.netlify.app/dev:864475044215258`
2. Click around on the site and make sure everything loads correctly
3. Settings are visible but not updatable without PIN included
4. Make sure quickstart docs look good

Any other sorts of testing notes to include?

Basically, all current functionality in production version of Airnote should be the same. This is just an internal upgrade.

## Any other related PRs

n/a

## Ticket(s)

https://trello.com/c/2xwdHO8P/598-update-airnote-project-to-sveltekit-20
